File: Facade/IPostalcodeService.vb
Imports BO
Public Interface IPostalcodeService
    Function GetPostalcodeById(PostalcodeId As Integer) As GetResponse(Of PostalCodeBO)
    Function GetPostalcodeByCountry(CountryId As Integer) As GetResponse(Of PostalCodeBO)
    Function GetPostalcodeByCountryAndSearchstring(CountryId As Integer, Searchstring As String) As GetResponse(Of PostalCodeBO)

    Function GetPostalcodeByCountryCodeAndPostalcode(Countrycode As String, Postalcode As String) As GetResponse(Of PostalCodeBO)
End Interface

File: FacadeCore/IPostalcodeService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using BOCore;

namespace FacadeCore
{
    public interface IPostalcodeService
    {
        GetResponse<PostalCodeBO> GetPostalcodeById(int PostalcodeId);
        GetResponse<PostalCodeBO> GetPostalcodeByCountry(int CountryId);
        GetResponse<PostalCodeBO> GetPostalcodeByCountryAndSearchstring(int CountryId, string Searchstring);
        //GetResponse<PostalCodeBO> GetPostalcodeByCountryCodeAndPostalcode(string Countrycode, string Postalcode);
    }
}
